action_server/docs/CHANGELOG.md

```diff
@@ -2,14 +2,14 @@
 
 ## Unreleased
 
-## 0.9.1 - 2024-05-17
-
-- Accept passing `OAuth2 secrets` (as a dict with the required keys) in the `/api/secrets`.
 - `new` command allows to choose a template from a predefined list now. Command will list available templates,
   and prompt user to choose one. 
   - `--template` argument has been added, allowing to specify a concrete template when running the command (when specified,
     user won't be prompted to choose a template from the list).
 
+## 0.9.1 - 2024-05-17
+
+- Accept passing `OAuth2 secrets` (as a dict with the required keys) in the `/api/secrets`.
```

Comment:
I was looking for this entry [here](https://github.com/Sema4AI/actions/blob/master/actions/docs/CHANGELOG.md) but I couldn't find it. What happened?

Reply:
You're looking into the changelog for actions... the proper one is the one from the action server: https://github.com/Sema4AI/actions/blob/master/action_server/docs/CHANGELOG.md